ui/desktop/src/components/ChatInput.tsx

```diff
@@ -841,6 +843,13 @@ export default function ChatInput({
     }
   };
 
+  const canSubmit =
```

Comment:
are diffs like this a side effect of things - as it doesnt' seem to have changed